package/yast2-printer.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed May 18 19:01:28 UTC 2016 - mchandras@suse.de
+
+- Remove /bin/ping dependency. All references to 'ping' have been
+  removed in 0ba65748c88b("apply jsmeix patches upstream"). This allows
```

Comment:
Please bump the version in the spec file (you can use `rake version:bump`) and mention the version also here. If the version is increased then our CI jobs automatically submit the package to Factory instead of keeping it just in the devel project.